### ISSUE TITLE
URL of the target language document overwritten by source language document

### ISSUE THREAD

**Issue by vitaka** [bitextor/bitextor#82]
There seems to be an error when Paracrawl release 3 TMX documents were built from the raw corpora.

After examining the en-mt TMX filtered with Bicleaner, it seems that all TUs have the same source documents in the SL and in the TL.

For example, consider the pair of sentences 
```
MarkizaZachodA: = false ;  
```
```
MarkizaZachodA: = foloz ;
```
in the en-mt raw document (downloaded from https://paracrawl.eu/releases.html). The URLs are:
```
http://home-automation.ehouse.pro/home-automation-such-as-creating-your-own-control-algorithms-for-system-ehouse/
```
```
http://home-automation.ehouse.pro/automation-home-bhall-holqien-algoritmi-tieghek-kontroll-sabiex-ehouse-sistema/
``` 

However, if we search the sentences in the Bicleaner-filtered TMX, we find the following XML fragment:
```
<tu tuid="4311081" datatype="Text">
    <prop type="score">2.3</prop>
    <prop type="score-zipporah">-0.2122</prop>
    <prop type="score-bicleaner">0.8157</prop>
    <prop type="lengthRatio">1.0</prop>
    <prop type="type">1:1</prop>
    <tuv xml:lang="en">
     <prop type="source-document">http://home-automation.ehouse.pro/home-automation-such-as-creating-your-own-control-algorithms-for-system-ehouse/</prop>
     <seg>MarkizaZachodA: = false ;</seg>
    </tuv>
    <tuv xml:lang="mt">
     <prop type="source-document">http://home-automation.ehouse.pro/home-automation-such-as-creating-your-own-control-algorithms-for-system-ehouse/</prop>
     <seg>MarkizaZachodA: = foloz ;</seg>
    </tuv>
   </tu>
```

**Comment by mespla** [comment]
Hi Víctor, yes, this was a bug in the script that builds TMX and was fixed in commit c960538c6f086ef02e828b1f706aa711adacafd3

**Comment by vitaka** [comment]
Nice, closing issue.
